### PR TITLE
Add native sarif output

### PR DESCRIPTION
**Pull request by yongyan-gh**
including CWE taxonomies support.

please review the attached sarif report generated using the functionality. [test.zip](https://github.com/yongyan-gh/flawfinder/files/6393859/test.zip)

### CODE REVIEW

File: flawfinder.py [hunk 1]
```diff
@@ -54,6 +54,7 @@
 import csv  # To support generating CSV format
 import hashlib
 # import formatter
+from sariflogger import SarifLogger
```
Comment (david-a-wheeler) [inline flawfinder.py:57]: Flawfinder doesn't use separate files, so that installation can be as simple as "copy this single file".

**Comment by david-a-wheeler** [comment]
I'd like it merged into 1 file (to ease copy-file installs), but that's a nit. I love having the new capability. So unless there's an objection, I plan to merge this soon, then pull it into 1 file.

Thank you so much for your hard work!!

**Comment by david-a-wheeler** [comment]
Thanks so much! As you can see, I've merged it.

It'd be great if you could create a test or two.

**Comment by eddynaka** [comment]
> I'd like it merged into 1 file (to ease copy-file installs), but that's a nit. I love having the new capability. So unless there's an objection, I plan to merge this soon, then pull it into 1 file.
> 
> Thank you so much for your hard work!!

Hi @david-a-wheeler , we thought about it (adding everything to one file). We separated to be easier to understand and in a separate "concern", only sarif related.

**Comment by david-a-wheeler** [comment]
I'd love to have a GitHub integration, and GitHub likes SARIF. If you were planning to create a GitHub Actions integrations, let me know!

**Pull request merged**